FHEM/14_SD_WS.pm

```diff
@@ -1546,6 +1591,23 @@ sub SD_WS_Parse {
   $def = $modules{SD_WS}{defptr}{$deviceCode} if(!$def);
 
   if(!$def) {
+    my @found = devspec2array("TYPE=SD_WS:FILTER=replaceBattery>0");
+    if (@found) {
+      if (($longids ne "0") && ($longids eq "1" || $longids eq "ALL" || (",$longids," =~ m/,$model,/))) {
+        my $rname = $found[0];
+        my $rhash = $defs{$rname};
+        my $rdef = $rhash->{DEF};
+        my $rproto = $rhash->{$ioname . '_Protocol_ID'};
+        if (defined $rproto && $rproto eq $protocol) {
+          Log3 $name, 3, "$ioname: SD_WS_Parse, $rname change DEF from $rdef in $deviceCode";
```

Comment:
Sollten wir hier nicht mit $rname ausgeben anstelle das IO Device?

Reply:
Ja, das sieht besser aus:
```
2022.02.06 14:00:09 3: SD_WS_33_T_4: SD_WS_Set, wait 300 seconds for a new identity of the sensor
2022.02.06 14:02:00 3: SD_WS_33_T_4: SD_WS_Parse, change DEF from SD_WS_33_T_245_1 in SD_WS_33_T_181_1
```

Reply:
Die Ausgabe habe ich auch geändert:

`          Log3 $name, 3, "$rname: SD_WS_Parse, change DEF from $rdef in $deviceCode";
`